TODO/6-practical-skills-for-ux-designers.md

```diff
@@ -88,10 +91,11 @@ Ji lee — Words As Image
 ![](http://ac-Myg6wSTV.clouddn.com/3ed1526bbd427508ad81.png)
 
 Ji lee — Words As Image
-For designers, it may look like hobbies and [side projects](http://pleaseenjoy.com/projects/personal/bubble-project/), or design explorations beyond the required. It is a optimistic outlook for what’s coming next.
 
-#### For me, the most valuable skill is the ability to learn from failure.
+对于设计师们来说，这看起来像是习惯或是一些[辅助项目](http://pleaseenjoy.com/projects/personal/bubble-project/)，或是需求之后的设计探索。其实这是对于将来的事情最佳的结果。
```

Comment:
> For designers, it may look like hobbies and [side projects](http://pleaseenjoy.com/projects/personal/bubble-project/), or design explorations beyond the required.

_hobbies_ 建议翻译成 “爱好”
_side project_ 建议翻译成 ”小项目“就可以了
_explorations beyond required_ 建议翻译成 ”超越基本要求的探索“